test/data/pti/parser_test_defaults.raw

```diff
@@ -0,0 +1,50 @@
+,,,,,
```

Comment:
We can use files in dependent packages without copying them over, we shouldn't add these files directly like this

Reply:
@frederikgeth I don't see where that file is being used, can I delete it?